src/sentry/explore/endpoints/explore_saved_queries.py

```diff
@@ -129,6 +129,12 @@ def get(self, request: Request, organization) -> Response:
         else:
             order_by = ["lower_name"]
 
+        exclude = request.query_params.get("exclude")
+        if exclude == "shared":
+            queryset = queryset.filter(created_by_id=request.user.id)
+        elif exclude == "owned":
+            queryset = queryset.exclude(created_by_id=request.user.id)
+
```

Comment:
Should we handle the case where exclude is not owned or shared?

Reply:
If exclude is neither `owned` or `shared`, the queryset just returns all saved queries